meta/boardreports/2022-11.md

```diff
@@ -0,0 +1,26 @@
+# Patterns Board Report (Nov)
+
+## Engagement
+
+The [patterns book] is the way inner source practices are captured and shared. Recent web analytics:
+
+* TODO: general trend in traffic (up/down?)
+* TODO: top 3 most popular patterns are? Did pattern of the month have any noticeable effect here?  
+* TODO: any insights from jp language traffic?
```

Comment:
```suggestion
* The Japanese version of the site has had 700 page views since its release and 450 unique users(As of November 2022). We need more page views in the future. Currently, many GitBook contents are not listed in Japanese search results. It must be search engine optimized.
```

Reply:
@yuhattor could you share a screenshot of how the search results for Japanese content look like right now?